content/doc/book/pipeline/syntax.adoc

```diff
@@ -13,17 +13,18 @@ title: Pipeline Syntax
 
 = Pipeline Syntax
 
-This section builds on the information introduced in <<getting-started#,
-Getting Started>>, and should be treated solely as a reference. For more
-information on how to use Pipeline syntax in practical examples, refer to
-<<jenkinsfile#, The Jenkinsfile>> section of this chapter. As of version 2.5 of
-the Pipeline plugin, Pipeline supports two discrete syntaxes which are detailed
-below. For the pros and cons of each, see the <<compare>>.
-
-As discussed in <<getting-started#, Getting Started>>, the most fundamental
-part of a Pipeline is the "step." Basically, steps tell Jenkins _what_ to
-do, and serve as the basic building block for both Declarative and Scripted
-Pipeline syntax.
+This section builds on the information introduced in
+link:../getting-started[Getting started with Pipeline] and should be treated
+solely as a reference. For more information on how to use Pipeline syntax in
+practical examples, refer to the
+link:../jenkinsfile[Using a Jenkinsfile] section of this chapter. As of version
+2.5 of the Pipeline plugin, Pipeline supports two discrete syntaxes which are
+detailed below. For the pros and cons of each, see the <<compare>>.
+
+As discussed at the link:../[start of this chapter], the most fundamental part
+of a Pipeline is the "step". Basically, steps tell Jenkins _what_ to do and
+serve as the basic building block for both Declarative and Scripted Pipeline
+syntax.
 
```

Comment:
Essentially, the only things that have changed above are link syntax, link wording and one page link itself (i.e. from the Getting started with Pipeline page to the Pipeline chapter page itself, since the latter is where the "step" is clearly described).